meta/service.go

```diff
@@ -0,0 +1,60 @@
+package meta
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/hashicorp/raft"
+	//"github.com/hashicorp/raft-boltdb"
+)
+
+// Service represents
+type Service struct {
+	raft *raft.Raft
+}
```

Comment:
The service should be fairly simple. It just passes through to Raft which will apply to the state machine using the commands.